tools/build_pytorch_libs.py

```diff
@@ -33,8 +32,12 @@ def _create_build_env():
     # you should NEVER add something to this list. It is bad practice to
     # have cmake read the environment
     my_env = os.environ.copy()
-    if USE_CUDA:
-        my_env['CUDA_BIN_PATH'] = CUDA_HOME
+    if 'CUDA_HOME' in my_env:  # Keep CUDA_HOME. This env variable is still used in other part.
+        my_env['CUDA_BIN_PATH'] = my_env['CUDA_HOME']
+    elif IS_WINDOWS:  # we should eventually make this as part of FindCUDA.
+        cuda_win = glob('C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v*.*')
+        if len(cuda_win) > 0:
+            my_env['CUDA_BIN_PATH'] = cuda_win[0]
```

Comment:
Wait, I just recognized that this value is not working. We should append '/bin' as suffix.